plugins/basic/nexus-repository-raw/src/main/java/org/sonatype/nexus/repository/raw/internal/RawGroupRecipe.groovy

```diff
@@ -89,7 +94,14 @@ class RawGroupRecipe
     Router.Builder builder = new Router.Builder()
 
     builder.route(new Route.Builder()
-        .matcher(new TokenMatcher("/{name:.+}"))
+        .matcher(new SuffixMatcher('/', false))
```

Comment:
the handler could still do that if it wanted, to cope with index.html and index.htm or whatever.

/me shrugs